modules/apps/transfer/keeper/relay.go

```diff
@@ -213,13 +213,13 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 				return err
 			}
 
-			unprefixedDenomPath := token.Denom.FullPath()
```

Comment:
I thought that `unprefixed` here could be a bit misleading. It's unprefixed with the port ID/channel ID of the sender, but if this is an IBC token with more trace, it is still prefixed by other port ID/channel ID pairs.